examples/user-notification.amp.html

```diff
@@ -229,6 +229,37 @@
      <button on="tap:amp-user-notification3.dismiss">I accept</button>
   </amp-user-notification>
 
+  <amp-user-notification
+          layout=nodisplay
+          id="amp-user-notification4"
+          always-show
+          data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
I feel that `persistent-dismissal` seems the closest to what we want, @cramforce go with that?